daemons/lvmdbusd/utils.py

```diff
@@ -281,12 +282,30 @@ def log_error(msg, *attributes):
 	_common_log(msg, *attributes)
 
 
+def dump_threads_stackframe():
+	id_to_name = {}
+	for i in threading.enumerate():
+		id_to_name[i.ident] = i
+
+	stacks = []
+	for tid, frame in sys._current_frames().items():
+		stack = traceback.format_list(traceback.extract_stack(frame))
+		stacks.append("Thread: %s" % (id_to_name[tid].name))
```

Comment:
This can easily fail as there is a race-condition between this code and any other code resulting in new threads being created. But maybe it would be enough to make this safer by using ``id_to_name.get(tid)`` and check if it returned something else than ``None``?

Reply:
Good point, I will make sure at the very least we won't throw an exception when we are going through this code and handle threads coming and going.